app/graphql/types/interface/media.rb

```diff
@@ -158,6 +158,18 @@ def reactions
     AssociationLoader.for(object.class, :media_reactions).scope(object)
   end
 
+  field :posts, Types::Post.connection_type, null: false do
+    description 'All posts that tag this media.'
+    argument :sort, Loaders::PostsLoader.sort_argument, required: false
+  end
+
+  def posts(sort: [{ on: :created_at, direction: :asc }])
+    Loaders::PostsLoader.connection_for({
+      find_by: :media_id,
+      sort: sort
```

Comment:
you'll need a `where: { media_type: type }` also to scope it correctly.

Reply:
Sorry, been out of it for a couple of days. Will look into this once I got time which might be later today or it might be on Sunday

Reply:
Fixed